meta-analysis/Mouse-Meta-Analysis.rmd

```diff
@@ -27,10 +27,35 @@ library(clustree)
 
 ## Load Mouse Atlas
 
-We will now load the mouse atlas into the R data section. This will be accomplished with the here library establishing the root of the project and then 
+The mouse atlas will now be loaded into the local R data section. This will be accomplished with the here library establishing the root of the project and then loading in the mouseAtlas.rds file into the mouseAtlas matrix object. A Seurat object will then be created in order to perform single cell analysis on the mouse atlas. 
 
-```{r pressure, echo=FALSE}
-plot(pressure)
+```r
```

Comment:
The code chunk syntax is wrong, it is being recognized as markdown syntax and formatting the code as text. To have it recognized as Rmarkdown and executed you need to surround the 

"\```r"   with   "```{r}"

Reply:
See commit 1e3f825